assets/js/googlesitekit/modules/datastore/modules.test.js

```diff
@@ -188,173 +215,46 @@ describe( 'core/modules modules', () => {
 			} );
 		} );
 
-		describe( 'recoverModule', () => {
```

Comment:
Deleting `recoverModule` tests while keeping the similar `recoverModules` tests have made this diff a bit hard to read. It might be a bit clearer taking a look through the commits instead.